### ISSUE TITLE
NTR: CXCL9 elevated in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8413]
New term request CXCL9 elevated in blood (dedeog_180717131028): CXCL-9; chemokine ligand  9

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000704
- label: Elevated circulating CXCL9 concentration
- parents: [Abnormality of serum cytokine level (HP:0011112)](https://hpo.jax.org/app/browse/term/HP:0011112)
- pmid: PMID:32060845
- def: The concentration of chemokine (C-X-C motif) ligand 9 in the blood circulation is above the upper limit of normal.
- comment: CXCL9, also known as monokine induced by gamma-interferon (MIG), can be produced during inflammatory conditions by myeloid cells within the tumor microenvironment.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8413
- synonyms: Elevated circulating monokine induced by gamma-interferon concentration(NONE)
